pkg/printers/name.go

```diff
@@ -65,15 +65,15 @@ func (p *NamePrinter) PrintObj(obj runtime.Object, w io.Writer) error {
 	groupVersionKind := obj.GetObjectKind().GroupVersionKind()
 	if len(groupVersionKind.Kind) > 0 {
 		if mappings, err := p.Mapper.RESTMappings(groupVersionKind.GroupKind(), groupVersionKind.Version); err == nil && len(mappings) > 0 {
-			fmt.Fprintf(w, "%s/%s\n", mappings[0].Resource, name)
+			fmt.Fprintf(w, "%s.%s/%s\n", mappings[0].GroupVersionKind.Kind, mappings[0].GroupVersionKind.Group, name)
```

Comment:
The whole point of this is remove the reliance on mapping.  You should be able to get the information from the obj, right?

Reply:
Thanks, addressed in latest commit